src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const carCost = days * 40;
+
+  if (days > 6) {
+    return carCost - 50;
```

Comment:
The discount logic for more than 6 days is correctly implemented. Ensure that this aligns with the task requirements.